Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs

```diff
@@ -53,9 +54,15 @@ void IOnDamageBehavior.OnDamageThresholdPassed(object obj, DamageThresholdPassed
             if (e.Passed && e.DamageThreshold == Threshold && destroyed == false)
             {
                 destroyed = true;
-                var wreck = Owner.EntityManager.SpawnEntity(spawnOnDestroy);
-                wreck.Transform.GridPosition = Owner.Transform.GridPosition;
-                Owner.EntityManager.DeleteEntity(Owner);
+                var coord = Owner.Transform.GridPosition;
+                var entMgr = Owner.EntityManager;
+                Owner.Delete();
+                if (string.IsNullOrWhiteSpace(spawnOnDestroy)) {
+                    Timer.Spawn(1, () =>
```

Comment:
After some more thinking I decided this fix was fine, although I would like a better way to specify that this timer should run "when the systems are done" instead of just putting a 1 ms delay.